### PR TITLE
store/tikv: log slow processing and waiting cop tasks separately with different labels

### PR DESCRIPTION
**Pull request by zz-jason**
- Use "TIME_COP_PROCESS" to log the slow cop tasks whose processing time is too long
- Use "TIME_COP_WAIT" to log the slow cop tasks whose waiting time is too long

### CODE REVIEW

**Comment by zz-jason** [comment]
@shenli @coocood PTAL

**Comment by ngaut** [comment]
We should add it to next release notes.

**Comment by coocood** [comment]
LGTM

File: store/tikv/coprocessor.go [hunk 1]
```diff
@@ -644,20 +644,22 @@ func (worker *copIteratorWorker) logTimeCopTask(costTime time.Duration, task *co
...
+				logStr = appendScanDetail(logStr, "write", detail.ScanDetail.Write)
+				logStr = appendScanDetail(logStr, "data", detail.ScanDetail.Data)
+				logStr = appendScanDetail(logStr, "lock", detail.ScanDetail.Lock)
 			}
-			if waitMs > minLogKVWaitTime {
-				logStr += fmt.Sprintf(" kv_wait_ms:%d", waitMs)
+		}
+		if waitMs > minLogKVWaitTime {
+			logStr += fmt.Sprintf(" kv_wait_ms:%d", waitMs)
+			if !(processMs > minLogKVProcessTime) {
```
Comment (ngaut) [inline store/tikv/coprocessor.go]: Use less or equal.

**Pull request merged**